app/services/SelfEmploymentService.scala

```diff
@@ -51,4 +53,11 @@ class SelfEmploymentService @Inject() (connector: SelfEmploymentConnector)(impli
     if (turnover > maxIncomeTradingAllowance) maxIncomeTradingAllowance else turnover
   }
 
+  def convertBigDecimalToMoneyString(amount: BigDecimal): String = {
+    val numberFormat  = NumberFormat.getNumberInstance(Locale.US)
```

Comment:
hmm, are you sure it should be US? that's what we use in other places? Not Locale.UK?